connectivity/tests/host.go

```diff
@@ -89,15 +96,25 @@ func (s *podToHostPort) Run(ctx context.Context, t *check.Test) {
 			echo := echo // copy to avoid memory aliasing when using reference
 
 			baseURL := fmt.Sprintf("%s://%s:%d%s", echo.Scheme(), echo.Pod.Status.HostIP, check.EchoServerHostPort, echo.Path())
+
+			var ipFamStr string
+
+			t.ForEachIPFamily(func(ipFam check.IPFamily) {
```

Comment:
Thanks for the PR. You have a syntax error here. This func here is never closed.

I'd recommend testing the code out locally too, that will make it easier to spot defects.

Reply:
Ah silly 😭  
Thank you so much @gandro for the catch! Because of the fact that I work on two different PC's (`Ubuntu 22.04 LTS` + `Windows 10`), the mistake happened while I was shifting the code from one system to another. 😅 

> And thanks for the advice, I'll make sure to test the code out locally too to avoid such silly mistakes. 😄

Reply:
Also, @gandro this → [`github.com/cilium/cilium-cli/connectivity/check`](github.com/cilium/cilium-cli/connectivity/check) is throwing 404.

Additionally, how can I check the IP then? (refer to [this](https://github.com/cilium/cilium-cli/pull/1440/commits/8bef04a0cf94c8904322524bcaf811d35bd0aae4#diff-ad5bbb6bf8573892c9d19da501655d19000d19a334327b2790e5d45af02cb4ecR41))

Reply:
The proper import path should be `"github.com/cilium/cilium-cli/connectivity/check"`. This is a Go *import path* and therefore not a valid GitHub URL. The GitHub URL for that file would be https://github.com/cilium/cilium-cli/tree/master/connectivity/check

> Should I have to use [this](https://github.com/cilium/cilium-cli/blob/master/connectivity/check/utils.go#LL14-L15C12)?

Yes, that's the one to use.